packages/module/src/AttachmentEdit/AttachmentEdit.test.tsx

```diff
@@ -51,4 +52,47 @@ describe('AttachmentEdit', () => {
     fireEvent.click(screen.getByText('Cancel'));
     expect(onCancelHandler).toHaveBeenCalled();
   });
+
+  it('should render custom button text for footer actions buttons', () => {
+    render(
+      <AttachmentEdit
+        code="Hello world"
+        fileName="greetings.txt"
+        isModalOpen={true}
+        onCancel={jest.fn()}
+        onSave={jest.fn()}
+        handleModalToggle={jest.fn()}
+        primaryActionButtonText="Save"
+        secondaryActionButtonText="Close"
+      />
+    );
+
+    screen.getByText('Save');
+    screen.getByText('Close');
+  });
```

Comment:
Can we just add a couple of `expect` lines to ensure these 2 texts are in the document?

Reply:
Updated